tests/test_synced_collection.py

```diff
@@ -8,11 +8,17 @@
 from tempfile import TemporaryDirectory
 from collections.abc import MutableMapping
 from collections.abc import MutableSequence
-from copy import deepcopy
+from copy import copy
 
 from signac.core.synced_list import SyncedCollection
 from signac.core.jsoncollection import JSONDict
 from signac.core.jsoncollection import JSONList
+from signac.core.zarrcollection import ZarrDict
+from signac.core.zarrcollection import ZarrList
+from signac.core.rediscollection import RedisDict
+from signac.core.rediscollection import RedisList
+from signac.core.mongodbcollection import MongoDict
+from signac.core.mongodbcollection import MongoList
```

Comment:
I would very much prefer to have the backend specific tests in their own respective test modules.